src/man/bhyve.5

```diff
@@ -180,10 +180,15 @@ presented to the guest.
 Available options are:
 .Bl -bullet
 .It
-.Sy virtio-net-viona Pq default
+.Sy virtio-net-viona Pq accelerated virtio interface, default
+.It
+.Sy virtio-net Pq legacy virtio interface
 .It
 .Sy e1000
 .El
+.Pp
```

Comment:
Should we also mention that only the legacy virtio-net will support promisc? That should help people that want to try and run things like VRRP/CARP/… in the guest.

Reply:
That isn't there yet. We need 13763 and 13738 first.

Reply:
So e1000 works without those too?

Reply:
No, you can attach them to the VM, and they will appear as PCI devices, but they will not work.

Reply:
Ack